### DISCUSSION TITLE
Custom Hard Disk, CDrom Drives Names

### DISCUSSION THREAD

**Discussion by ItzZisker** [86Box/86Box#1255] [Ideas]
i just wanted to know there are a feature to change the name of cdrom or hard drives to something different and make it more realistic? it is possible? for example changing 86BOX-HD to WD-WMAO22993555. if it didn't how can i change it in source code and where the code written about the name of the hard disk drive? can someone help, maybe if this feature does not exist, you can add that, it makes emulation more realistic.

**Comment by ppgrainbow** [comment]
This would be a pretty neat idea. But I don't know when it's going to be implemented.

Currently that feature only exists in Bochs, which hasn't been updated in over a year.
